yarn.lock

```diff
@@ -4269,6 +4269,11 @@ levn@~0.3.0:
     prelude-ls "~1.1.2"
     type-check "~0.3.2"
 
+libphonenumber-js@^1.9.53:
```

Comment:
Ditto an [earlier note](https://github.com/18F/identity-idp/pull/6306#discussion_r865826297) about deduplicating to get rid of this extra dependency, since otherwise we'll be shipping two different versions to users.

```
yarn global add yarn-deduplicate
yarn-deduplicate
```